config.tfvars

```diff
@@ -57,6 +57,13 @@ jira_replica_count = 1
 # the following line and set the jira_version_tag to any of the versions available on https://hub.docker.com/r/atlassian/jira-software/tags
 #jira_version_tag = "<JIRA_VERSION_TAG>"
 
+# To select a different image repository for the Jira application, you can change following variable:
+# Official suitable values are:
+# * "atlassian/jira-software"
+# * "atlassian/jira-servicemanagement"
+# * "atlassian/jira-core"
```

Comment:
nit: It's not listed in JIRA_CONFIGURATION.md, maybe we can drop `jira-core` here, given `jira-core` does not normally being installed independently anyway.